src/services/Auth/RegistrationApi.js

```diff
@@ -0,0 +1,81 @@
+import api from "Services/Api";
+import Auth from "Services/Auth/Auth";
+
+async function registerEmail(email = "") {
+  return api.post("/users/", { email });
+}
+
+async function verifyEmail(user_id = "", token = "") {
+  const response = await api.post(`/users/${user_id}/verify/`, {
+    confirmation_token: token,
+  });
+  if (response) {
+    api.setToken(response.token);
+    Auth.setAuthenticated();
+  }
+}
+
+async function logout(callback) {
+  await api.reset();
+  await Auth.setNotAuthenticated();
+  if (callback) callback();
+}
+
+async function setPassword(userId = "", newPassword = "") {
+  const data = {
+    password: newPassword,
+  };
+  return api.post(`/users/${userId}/set_password/`, data);
+}
+
+async function createProfile(user = null, fname = '', lname = '', birthDate = '', phoneNum = '') {
```

Comment:
profile shouldn't be in Registration (registration should only be auth stuff), you should create a new Api file for profile